README.md

```diff
@@ -1,53 +1,34 @@
-
-The Shopify theme gem is a command line tool that lets you make live changes to themes on your Shopify store. If the command line is scary, check out the [Desktop Theme Editor app](http://apps.shopify.com/desktop-theme-editor).
-
-It will watch your local folders for any changes in your theme (including adding and removing files) and will update your .myshopify.com store to the latest changes. 
+# Edit Your Shopify Theme Locally
 
 ![Shopify theme gem](https://dl.dropboxusercontent.com/u/669627/terminalreadme.png)
 
-You do not need to make changes to your default theme. You can leverage the theme preview feature of Shopify
-that allows you to view what an unpublished theme looks like on your Shopify store. This means you don't need to
-sign up for extra accounts and keep your shop up to date. You will however have a blue bar that shows up that you can
-remove via the web inspector in Chrome or Safari.
-
-# Requirements
+The Shopify theme gem is a command line tool that allows you to manage and edit theme files directly on your computer. This allows you to use your preferred code editing software opposed to the built in code editor available in the “Edit HTML/CSS” page of your store. Files can be manually or automatically uploaded, downloaded, and synced. Depending on how you develop themes and your personal preference, this can allow for a more enjoyable and streamlined workflow.  If the command line is scary for you, check out the [Desktop Theme Editor app](http://apps.shopify.com/desktop-theme-editor).
 
-This gem works with OS X or Windows with Ruby 1.9. 
 
-First time installing Ruby on windows? Try [Rubyinstaller](http://http://rubyinstaller.org/). 
+## Requirements
+This gem works with OS X or Windows with Ruby 1.9.
+First time installing Ruby on windows? Try Rubyinstaller.
 
-# Installation
 
-To install the shopify_theme gem use 'gem install' (you might have use 'sudo gem install')
-
-```
-gem install shopify_theme [optional_theme_id]
-```
+## Configuration
+### Setting up Shopify Theme
 
-to update to the latest version
+First, you will need to set up a new private app to generate an API key and password. Go to [your-store-url].com/admin/apps/private in your web browser.
 
-```
-gem update shopify_theme
-```
+Click on “Create a Private App” to generate the credentials for a new app. You will need the API key and the password of this newly generated app:
 
-# Usage
+![api-key-and-password](https://cloud.githubusercontent.com/assets/5488872/5901847/958068e4-a543-11e4-8eb4-8f139cc4384c.jpg)
 
-Generate the config file. Go get a valid api_key and password for your store head to `https://[your store].myshopify.com/admin/apps/private` and generate a private application. By default it adds the main theme, if you want to edit one of your other themes, add the `theme_id`.
+In the command line, go to the directory of your theme files, or the directly where you would like your theme files to be.
```

Comment:
This entire section reads kinda weirdly. There's lots of repetition so it's awkward. What about something like:

--------

Navigate to the directory where you theme files live, or where you'd like them to be and execute the following command:

 `theme configure api_key password store_name`

In your directory you should see a file named `config.yml` with your credentials. If you want you may also edit the `config.yml` file and paste your API Key and Password in the appropriate area.